native-addons-are-close-to-being-on-par-with-js-modules.md

```diff
@@ -0,0 +1,113 @@
+# Native Addons Are Close To Being On Par With JS Modules
+
+Native addons have been a part of Node.js since the very beginning. So have JS modules 😉 So, what does it mean that they are now close to being "on-par"? Why only now?
+
+Let's first look at the similarities between native adons and JS modules. Both can be `require()`-ed:
+
+```JS
+// A JS module can be loaded from a package:
+const jsModulePackage = require('my-js-module');
+// It can also be loaded from a specific file:
+const jsModuleFile = require('./my-js-module.js');
+
+// A native addon can also be loaded from a file:
+const nativeAddonFile = require('./build/Release/my-native-addon.node');
+```
+
+Both return a JavaScript object containing useful functionality which would otherwise be completely unavailable, or which would otherwise take a long time to implement.
+
+Nevertheless, the similarities end there. Let's now contrast the two types of modules:
+
+|  |  | JavaScript modules | Native addons |
+|--- |--- |--- |--- |
+| 1. | ... need to be compiled. | No | Not if there are pre-builds |
+| 2. | ... will work on all platforms and architectures. | Yes | Yes if there are pre-builds |
+| 3. | ... will work with all current and subsequent Node.js versions once written. | Yes | No |
```

Comment:
it is designed to work with all current and subsequent Node.js versions once written - should be more correct. Maybe future designs or R&D will bring more sub-categorizations, so some things might not necessarily work in the future, as we see it now.